### DISCUSSION TITLE
VTM Blue grid

### DISCUSSION THREAD

**Discussion by troNpo** [mapsforge/vtm#1153] [Q&A] [answered]
Hello, in the OruxMaps GP app, when using the vtm viewer with Mapsforge maps (Openandromaps) and Themes such as Elevate.
Blue grids are shown in the background of the map.
![VTM_barras2](https://github.com/user-attachments/assets/5ffd4b22-7dbd-4a9b-84d8-8c362c8a7b16)
I have checked that these Grid  are displayed on Mapsforge v5 and V4 maps, but not on v3 (OSM Paws).
Is it normal behavior with this Viewfinder ?
The grids appear to match the DEM boundaries, but even without having these (DEM) files in the app, they are displayed.
Thank you.

**Comment by devemux86** [comment, marked as answer]
@troNpo 

VTM samples all this time seem to be working properly.

So it may be a problem on how the app uses the library.
For this, you can ask in the forum of the app.

---

However, VTM and OpenGL have various rendering problems.

Better use [Mapsforge](https://github.com/mapsforge/mapsforge), now it has map rotation and new hill shading.

**Reply by troNpo** [reply]
Thanks for the check, I will pass the problem on to the OruxMaps developer.
If I know the improvements of the Mapsforge libraries, oruxmaps also uses the new hill shading.
Thank you very much for everything.
